.travis.yml

```diff
@@ -1,58 +0,0 @@
-language: python
-
-os:
-  - linux
-
-python:
-  - 2.7
-  - 3.4
-  - 3.5
-  - 3.6
-  - 3.7
-  - 3.8
-
-matrix:
-  include:
-    # Travis does not support Python versions on OSX:
-    # https://github.com/travis-ci/travis-ci/issues/2312
-    - language: generic
-      os: osx
-      # Set the Python version to display it in the Travis UI.
-      python: 2.7
-    - language: generic
-      os: osx
-      # Set the Python version to display it in the Travis UI.
-      python: 3.6
-      before_install:
-        - if ! command -v python3; then
-            brew update;
-            brew install python3;
-            virtualenv env -p python3;
-            source env/bin/activate;
-          fi
-    # Test manylinux1 64 and 32 bit wheels.
```

Comment:
Can you open an issue for testing macos and manylinux?

Reply:
Yes. https://jira.mongodb.org/browse/PYTHON-3023